src/hooks/useCheckRevision.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import useFocusInput from './useFocusInput';
+
+const useCheckRevision = () => {
+  const { handleChildClick } = useFocusInput();
+  const [checked, setChecked] = React.useState<string[]>([]);
+
+  const handleToggle = (e: React.MouseEvent) => {
+    const revisionID = (e.currentTarget as HTMLElement).id;
+    const currentIndex = checked.indexOf(revisionID);
+    const newChecked = [...checked];
+
+    handleChildClick(e);
+
+    if (checked.length < 4 && currentIndex === -1) {
+      newChecked.push(revisionID);
+    } else if (currentIndex !== -1) {
+      newChecked.splice(currentIndex, 1);
+    } else {
+      // TODO: replace with pop up notification
+      console.log('maximum four revisions');
```

Comment:
[question] shld this removed ?

Reply:
This is temporary, I've replaced this with a modal in #113